aristamd/src/style/aristamd/app.accordion.less

```diff
@@ -0,0 +1,89 @@
+.accordion__content {
+    background: @body-bg;
+    overflow: auto;
+    padding: 10px 30px 10px;
+}
+.animate-show-hide {
+  transition:0.2s all ease-in-out;
+  -webkit-transition-delay: 0.1s; /* Safari */
+  transition-delay: 0.1s;
+  opacity:1;
+}
+.animate-show-hide.ng-hide {
+  opacity:0;
+  -webkit-transition-delay: 0.01s; /* Safari */
+  transition-delay: 0.01s;
+}
```

Comment:
Let's try to use more encapsulation.
Instead of:
```
.animate-show-hide {
  transition:0.2s all ease-in-out;
  -webkit-transition-delay: 0.1s; /* Safari */
  transition-delay: 0.1s;
  opacity:1;
}
.animate-show-hide.ng-hide {
  opacity:0;
  -webkit-transition-delay: 0.01s; /* Safari */
  transition-delay: 0.01s;
}
```
we could use:
```
.animate-show-hide {
  transition:0.2s all ease-in-out;
  -webkit-transition-delay: 0.1s; /* Safari */
  transition-delay: 0.1s;
  opacity:1;

  &.ng-hide {
    opacity:0;
    -webkit-transition-delay: 0.01s; /* Safari */
    transition-delay: 0.01s;
  }
}